### DISCUSSION TITLE
Just read the BMS (no ESPHome)

### DISCUSSION THREAD

**Discussion by AimbotNooby** [syssi/esphome-jk-bms#70] [Q&A] [answered]
Hey, because this is the only program I found to read the JK-BMS, I wonder if you can tell me how I can easily read the sensors without this ESPHome stuff (never worked with it) or if you have a solution to read the RS485 with a Raspberry Pi.
Btw thx for your efforts.

**Comment by jonadis** [comment, marked as answer]
ESPHome is awesome and you can use it without HomeAssistant if desired... 
ESPHome would, for example, give you a web page that shows BMS info that you could query via API as well if desired.
Alternatively, you can use [mpp-solar](https://github.com/jblance/mpp-solar) to read the BMS info on a Raspberry Pi.

**Reply by AimbotNooby** [reply]
Thank you that helped :)

**Comment by syssi** [comment]
If you own a ESP8266 or ESP32 we could guide you to a working setup. `mpp-solar` is the way to go if you don't want to use an ESP in your setup.

**Reply by AimbotNooby** [reply]
Alright thank you. Maybe I will come back in a certain time since I don't have a BMS or ESP right know.
